wp-content/themes/core/pcss/utilities/mixins/_buttons.pcss

```diff
@@ -1,45 +1,82 @@
 
 /* -----------------------------------------------------------------------------
  *
- * Global Button Resets
+ * Button Global Reset
  *
  * ----------------------------------------------------------------------------- */
 
 @define-mixin button-global {
-	transition: var(--transition);
-	width: auto;
+	border: 0;
+	cursor: pointer;
 	display: inline-block;
-	line-height: var(--line-height-even);
-	font-weight: var(--font-weight-semibold);
 	height: auto;
-	cursor: pointer;
-	text-decoration: none;
-	border: 0;
 	padding: 0;
+	text-decoration: none;
+	width: auto;
 }
 
 /* -----------------------------------------------------------------------------
- *
- * Button Submit
- *
+ * Button - Base
  * ----------------------------------------------------------------------------- */
 
-@define-mixin button-submit {
-	@mixin button-global;
-
-	background-color: var(--color-black);
-	padding: 1em;
+@define-mixin button-base {
```

Comment:
Given we have a `button-global` might provide greater naming clarity to have this be `button`?